extensions/tiled-assets/README.md

```diff
@@ -0,0 +1,133 @@
+# Tiled Assets specification
+
+- **Title: Tiled Assets**
+- **Identifier: tiled-assets**
+- **Field Name Prefix: tiles**
+- **Scope: Item**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+Some data products are too big to be handled in a single file or a small set of enumerated files and are thus split into tiles. For example, Sentinel-2 datastrips are tiled into overlapping granules, in some cases in even more than one coordinate reference system. Other very big datasets, such as continental or global mosaics can also only be handled in a tiled fashion. Usually, they go one step further and provide multiple layers of resolution to allow a quick inspection of larger areas but also retain the possibility to get to the full resolution data.
+
+When a very large dataset is split into many tiles it is very impractical to list all possible files with the `assets` property.
+
+This extension allows the specification of tiled assets within STAC Items. With this extension it is possible to allow the description of assets using template references and rules to construct those in order to get the tiles the user is interested in. The new `asset_templates` property allows to specify template URLs where components can be replaced to get the final URLs to the actual files.
+
+This extension is modelled in close alignment to the [OGC Two Dimensional Tile Matrix Set](http://docs.opengeospatial.org/is/17-083r2/17-083r2.html). The main access approach and the templating mechanism used is defined in the [OGC WMTS Simple Profile](http://docs.opengeospatial.org/is/13-082r2/13-082r2.html).
+
+- Examples: [Tiled](examples/example-tiled.json), [Dimension](examples/example-dimension.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Item, Collection and Catalog properties
```

Comment:
Here is says that the field applies to Catalogs and Collections, too, but the scope and JSON schema only mention items?! 🤔

Reply:
Oh, yes, good point. The `tile_matrix_sets` can be located at Collection/Catalog level. So I'll add this in the readme and schemas as-well.

Everything else would remain on Item level.